setup.py

```diff
@@ -10,6 +10,6 @@
     author_email='',
     # REPLACE WITH YOUR OWN GITHUB PROJECT LINK
     url='https://github.com/annikabrundyn/da_vinci',
-    install_requires=['pytorch-lightning'],
```

Comment:
@annikabrundyn should I add this back in?

Reply:
nope I don't think we need this